wp-cli/media-regenerate.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Helper for dealing with the media regenerate command
+ */
+namespace Automattic\VIP\Commands;
+
+use WP_CLI;
+
+$runner     = WP_CLI::get_runner();
+$assoc_args = $runner->assoc_args;
+
+$media_regenerate_before_invoke = function() use ( &$runner, &$assoc_args ) {
+	// If skip-delete is not set or not true
+	if ( ! isset( $assoc_args['skip-delete'] ) || true !== $assoc_args['skip-delete'] ) {
+		// add skip-delete to the assoc_args array
+		$assoc_args = wp_parse_args(
+			$assoc_args,
+			[ 'skip-delete' => true ]
+		);
+
+		WP_CLI::line( 'Forcing --skip-delete  flag as it\'s required for VIP File Service' );
```

Comment:
Extra space in message:

```suggestion
		WP_CLI::line( 'Forcing --skip-delete flag as it\'s required for VIP File Service' );
```